ansible/roles/start_vm/tasks/execute.yml

```diff
@@ -0,0 +1,52 @@
+- name: check if socket exists
+  stat:
+    path: /tmp/firecracker.socket
+  register: socket_exists
+
+- name: remove socket if exists
+  file:
+    state: absent
+    path: /tmp/firecracker.socket
+  when: socket_exists.stat.exists == True
```

Comment:
not needed. "Note that absent will not cause file to fail if the path does not exist as the state did not change"